app/src/test/java/org/glucosio/android/activity/HelloActivityTest.java

```diff
@@ -98,6 +115,30 @@ public void ShouldSelectLanguage_WhenCreated() throws Exception {
         assertThat(activity.languageSpinner.getSpinner().getSelectedItem()).isEqualTo("Nederlands");
     }
 
+    @Test
+    public void ShouldShowWebView_WhenTermsOfUseButtonIsClicked() throws Exception{
+        activity.onTermsAndConditionClick();
+        ShadowActivity shadowActivity = shadowOf(activity);
+        Intent startedIntent = shadowActivity.getNextStartedActivity();
+        assertEquals(ExternalLinkActivity.class.getName(), startedIntent.getComponent().getClassName());
```

Comment:
We use `AssertJ` library, our assertions look like `assertThat(something).isEqualTo(expected)`